open-hackathon-server/src/hackathon/template/template_unit.py

```diff
@@ -36,3 +36,12 @@ def __init__(self, provider):
         provider should be value of enum VE_PROVIDER.
         """
         self.provider = provider
+
+    def get_name(self):
+        raise NotImplemented
```

Comment:
Cannot catch you about the newly added three unimplemented methods. Please tell me more details